src/components/sendV2/transactionStatus/transactionStatus.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { PrimaryButtonV2 } from '../../toolbox/buttons/button';
+import Piwik from '../../../utils/piwik';
+import svg from '../../../utils/svgIcons';
+import styles from './transactionStatus.css';
+
+class TransactionStatus extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.backToWallet = this.backToWallet.bind(this);
+    this.onErrorReport = this.onErrorReport.bind(this);
+  }
+
+  backToWallet() {
+    Piwik.trackingEvent('TransactionStatus', 'button', 'Back to wallet');
+    // istanbul ignore else
+    if (this.props.failedTransactions !== undefined) this.props.transactionFailedClear();
+    this.props.finalCallback();
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  onErrorReport() {
+    const recipient = 'hubdev@lisk.io';
+    const subject = `User Reported Error - Lisk Hub - ${VERSION}`; // eslint-disable-line no-undef
+    return `mailto:${recipient}?&subject=${subject}`;
+  }
+
+  render() {
+    const isTransactionSuccess = this.props.failedTransactions === undefined;
+
+    let transactionStatus = {
+      headerIcon: svg.transactionSuccess,
+      bodyText: {
+        title: 'Transaction submitted',
+        paragraph: 'You will find it in My Transactions in a matter of minutes',
+      },
+    };
+
+    // istanbul ignore else
+    if (!isTransactionSuccess) {
+      transactionStatus = {
+        headerIcon: svg.transactionError,
+        bodyText: {
+          title: 'Transaction failed',
+          paragraph: 'Oops, looks like something went wrong. Please try again.',
+        },
+      };
+    }
+
+    return (
+      <div className={`${styles.wrapper} transaction-status`}>
+        <header className={styles.header}>
+          <img src={transactionStatus.headerIcon}/>
+        </header>
+        <div className={`${styles.content} transaction-status-content`}>
+          <h1>{this.props.t('{{title}}', { title: transactionStatus.bodyText.title })}</h1>
```

Comment:
the `this.props.t` should be on the assignment of the value: 
https://github.com/LiskHQ/lisk-hub/blob/a5d87bff9c1d8cd835800f500999abdb4fd2fdba/src/components/sendV2/transactionStatus/transactionStatus.js#L32-L49

Here since you are just interpolation, the variable won't be translated.

Reply:
I totally change this in the PR related HW as with the HW I had to add a new texts to the component so I totally redisign this.